README.md

```diff
@@ -4,6 +4,14 @@
 
 On systems with /dev/vcsu* (e.g. Raspbian Buster) and when using a TrueType font, the `terminal` mode now has full support for Unicode output. This is automatic, with a fallback to 8-bit if either of these requirements isn't met. We've also changed the standard encoding for 8-bit to ISO-8859-1, which should be a little closer to what the `vcs` device provides.
```

Comment:
The kernel's changelog (https://cdn.kernel.org/pub/linux/kernel/v4.x/ChangeLog-4.19) indicates that support for the `vcsu` was added in 4.19 and I'm still running Stretch (9.11), so this would be more accurate.

```suggestion
On systems with /dev/vcsu* (e.g. Raspbian Stretch or Buster, with kernel 4.19+) and when using a TrueType font, the `terminal` mode now has full support for Unicode output. This is automatic, with a fallback to 8-bit if either of these requirements isn't met. We've also changed the standard encoding for 8-bit to ISO-8859-1, which should be a little closer to what the `vcs` device provides.
```